index.js

```diff
@@ -19,11 +20,24 @@ function generateMarker (options, callback) {
     var basePaths = backgroundMarkerSize.svg.g[0].g[0].g;
 
     if (options.symbol) {
-        var symbolSize = `${options.symbol}-${size === 's' ? '11' : '15'}`;
-        if (!assets.parsedSVGs[symbolSize]) return callback(errcode(`Symbol ${options.symbol} not valid`, 'EINVALID'));
+        const isMaki = (symb) => symb.length > 2;
+
+        const symbolName = (isMaki(options.symbol))
+            ? options.symbol
+            : `${options.symbol}-${numericSizes[size]}`;
+
+        if (!assets.parsedSVGs[symbolName]) {
+            return callback(errcode(`Symbol ${options.symbol} not valid`, 'EINVALID'));
+        }
+
+        // because maki@7.0.0 removed small SVGs, we need to update our small-marker SVG
+        // to reduce the size of the maki icon it renders by the appropriate ratio
+        if (isMaki(options.symbol) && size === 's') {
+            basePaths[3]['$'].transform += ' scale(0.733, 0.733)';
+        }
```

Comment:
oof this is slightly annoying. Curious what size usage is like and whether that should be deprecated in a future version of the API or changed to a `scale` parameter for dynamic sizing.

Reply:
Ugh, yeah. I briefly thought about this and then banished the thought because this PR was supposed to just be a quick bug fix. 

I know anecdotally that the downstream dependencies definitely do make frequent use of the the size param, but I'd probably want dig for more data about it before running with a change to the API. 

I don't think we'd ever be able to remove the sizing options entirely, but exposing `scale` instead of a hardcoded size value is very appealing and could be worth a future update to the API.